views/js/qtiCreator/widgets/interactions/mediaInteraction/states/Question.js

```diff
@@ -29,9 +29,8 @@ define([
     'taoQtiItem/qtiCreator/widgets/helpers/formElement',
     'tpl!taoQtiItem/qtiCreator/tpl/forms/interactions/media',
     'ui/mediaEditor/mediaEditorComponent',
-    'ui/resourcemgr',
-    'ui/tooltip'
-], function ($, _, __, stateFactory, Question, formElement, formTpl, mediaEditorComponent) {
+    'util/urlParser'
```

Comment:
Are you using `'util/urlParser'`?
Please, return removed imports,  they are used in code

Reply:
i will remove the **url/parser** But i removed the imports because i cannot see a parameter for the same on function , in that case i thought the import was not directly used in the page , but i dont know it is being used indirectly , Can you confirm?